astropy/coordinates/builtin_frames/altaz.py

```diff
@@ -51,12 +51,12 @@ class AltAz(BaseCoordinateFrame):
         If `True` (default), make copies of the input coordinate arrays.
         Can only be passed in as a keyword argument.
 
-    differential_cls : `BaseDifferential`, dict, optional
+    differential_type : `BaseDifferential`, dict, optional
```

Comment:
Strange that the `str` option was missing again. See comment for `Baseframe` in that it also should be `BaseDifferential` subclass. I guess this holds for all other frames too. Sigh.

It may be worth using `astropy.utils.decorators.format_doc`...

Reply:
OK, I sucked it up and overhauled all of the docstrings to use `format_doc`. I did it in a single commit, so it's probably easiest to look at the diff for that commit...